sys/shell/commands/sc_interactive_sync.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2019 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_shell_commands
+ * @{
+ *
+ * @file
+ * @brief       Shell commands interactive sync util
+ *
+ * @author      Francisco Molina <francois-xavier.molina@inria.fr>
+ * @}
+ */
+
+#include <stdio.h>
+#include "test_utils/interactive_sync.h"
+
+int _s(int argc, char **argv)
```

Comment:
not sure `_s` or `_r` should be global symbols. IIRC symbols starting with single `_` are supposed to be file scope unique (and thus shouldn't be global).

Reply:
Are you suggesting to change the names all shell command functions in RIOT?
https://github.com/RIOT-OS/RIOT/blob/dd0d5f7fcf90411ca798edcfc839a5a09ddffce8/sys/shell/commands/sc_gnrc_netif.c#L1293

The function names could be a little more explicit: `_test_start`, `_test_ready`.